packages/related-keyphrase-suggestions/src/elements/TrendGraph/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+
+/* Internal dependencies */
+import TrendGraphTable from "./TrendGraphTable";
+
+/**
+ * Renders a SVG area chart.
+ *
+ * @param {array}    data                      Array of objects with X and Y coordinates for the SVG chart points.
+ * @param {number}   width                     The SVG chart width.
+ * @param {number}   height                    The SVG chart height.
+ * @param {string}   fillColor                 The SVG chart area background color in HEX format.
+ * @param {string}   strokeColor               The SVG chart line color in HEX format.
+ * @param {number}   strokeWidth               The SVG chart line width.
+ *
+ * @returns {JSX.Element} The SVG area chart component.
+ */
+const TrendGraph = ( {
+	data,
+	width = 66,
+	height = 24,
+	fillColor = "#ade3fc",
+	strokeColor = "#498afc",
+	strokeWidth = 1.8,
```

Comment:
Can't this go via styling/Tailwind classes?

Where with other elements you have almost no API, this is suddenly very changeable from props.

Ok, maybe not the `strokeWidth` as there are calculations below.

Reply:
Yes I agree, This is me copying the original component to the package, I was also thinking about that, I will remove those props